docs/providers.rst

```diff
@@ -1486,6 +1486,54 @@ The OpenID provider can be forced to operate in stateless mode as follows::
                     stateless=True,
                 )]}}
 
+
+OpenID Connect
+--------------
+
+The OpenID Connect provider provides a dynamic instance for each configured
+server. To expose an OpenID Connect server as an authentication method,
+configuration of one or more servers is required:
+
+.. code-block:: python
+
+    SOCIALACCOUNT_PROVIDERS = {
+        "openid_connect": {
+            "SERVERS": [
+                {
+                    "id": "my-server",  # 15 characters or less
```

Comment:
**Issue**: By dropping the prefix, this 15 characters remark and the one below can be bumped to 30.

Reply:
I updated both comments to say 30 characters or less.